src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentDay = 40;
+  const total = rentDay * days;
+
+  if (days >= 7) {
+    return total - 50;
+  }
+
+  if (days >= 3) {
+    return total - 20;
```

Comment:
Use a constant variable for the discount amount here as well (e.g. `const smallDiscount = 20;`), to make the code more maintainable and readable 👍